github3/events.py

```diff
@@ -30,6 +33,9 @@ class Event(GitHubCore):
     """
 
     def _update_attributes(self, event):
+        # The _json_data is modified here
+        self._orig_json_data = copy.deepcopy(event)
```

Comment:
Looks like `payload` is inserted into `__dict__` when `self.payload = ...` occurs. So `__getattr__` won't be called in that case.